app/services/messages.js

```diff
@@ -0,0 +1,30 @@
+import Ember from "ember";
```

Comment:
Good call with a service, we need as much logic to be centralized 👍

Reply:
Actually, if there's other pieces of code that can be added to the service, that'd be a good way to go

Reply:
sure